src/components/Error-404/Error404.jsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from "react-router-dom";
+import ErrorImage from "../../assets/error-404.png";
+
+export function Error404() {
+  return (
+    <main className="txt-center mt-2 p-1">
+      <figure>
+        <img src={ErrorImage} alt="Error" width="369" height="265" />
+      </figure>
+      <p>
+        Unfortunately the page you are looking for has been moved or deleted
```

Comment:
suggestion - This message can be changed to "Unfortunately the page you are looking for, does not exist!"

Reply:
M

> Code is clean and good to merge
> 
> Suggestion - you may add some more space between "go to homepage" and the footer

More margin is added to the button

Reply:
> Unfortunately the page you are looking for, does not exist!

Changed, thank you for the suggestions